tests/qpu_calibration_data/spam_twirling_mock.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Maybe add a comment describing what is in the file? And how it is used?

Looking at it I don't really know what the register is!

Reply:
added a README in the calibration data folder